WalletWasabi/WabiSabi/Backend/Statistics/CoinJoinFeeRateStatRecord.cs

```diff
@@ -0,0 +1,23 @@
+using NBitcoin;
+
+namespace WalletWasabi.WabiSabi.Backend.Statistics;
+
+public record CoinJoinFeeRateStatRecord(
```

Comment:
This is the only `record` where we use the word `Record` as a suffix. Maybe rename it to `CoinJoinFeeRateStat`?